### ISSUE TITLE
Feature request: net_version and eth_chainId should be configurable for private test networks

### ISSUE THREAD

**Issue by skylenet** [emeraldpay/dshackle#124]
Hi all! First of all, congrats on creating this amazing tool :) I've just started using it within some private networks that I'm running. 

These private networks use their own chain ID's and unfortunately it's not possible to configure dshackle with that ID. This is kinda breaking some consensus layer clients that expect the chain ID to be the same as they have in their genesis information. 

It would be great if I could provide a chainId via configuration and that this would be then returned when calling `net_version` and `eth_chainId`.

For now they seem to be just hardcoded: https://github.com/emeraldpay/dshackle/blob/5167b8b125a47a15e9037ebf8b7cc7be771625d3/src/main/kotlin/io/emeraldpay/dshackle/upstream/calls/DefaultEthereumMethods.kt#L116 

Maybe something like:

```yaml
# Content of dshackle.yaml
  proxy:
    host: 0.0.0.0
    port: 8545
    routes:
      - id: eth
        blockchain: ethereum
        chainid: 1333333
 ``` 
 
 Basically, if the chainid would be defined within the route, then it would use that instead of the hardcoded values.

**Comment by splix** [comment]
Hi. Thank you for the request. It's indeed a needed feature, and I already have an issue with it #96. However, I cannot say when I'll find a time to work on it. But if you think you can contribute towards that feature, I'd appreciate it.

Closing this one as a duplicate to #96
